core/templates/dev/head/library/library.html

```diff
@@ -59,10 +59,15 @@ <h2 class="oppia-library-h2" translate="I18N_LIBRARY_SUB_HEADER"></h2>
           <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index">
             <div class="row oppia-library-group-header-container">
               <h2 class="oppia-library-group-header">
-                <a ng-click="showAllResultsForCategories(group.categories)" translate="<[group.translationId]>">
+                <a ng-if="libraryGroups[$index].header != 'Featured Activities'"
```

Comment:
Thanks!

How hard do you think it would be to pass the i18n id to the frontend as a header, instead of the "Featured Activities" string? I have a feeling this might trip up future developers -- it wasn't obvious to you, me, or Xinyu, so that's a big red flag.